chickadee/processes/wps_bccaq.py

```diff
@@ -0,0 +1,147 @@
+from pywps import Process, ComplexOutput, LiteralInput, FORMATS
+from pywps.app.Common import Metadata
+from netCDF4 import Dataset
+import os
+
+from wps_tools.utils import log_handler
+from wps_tools.io import log_level
+from chickadee.utils import logger, set_r_options, get_ClimDown, get_doParallel
+
+
+class BCCAQ(Process):
+    """Bias Correction/Constructed Analogues with Quantile mapping reordering:
+    Full statistical downscaling of coarse scale global climate model (GCM)
+    output to a fine spatial resolution"""
+
+    def __init__(self):
+        self.status_percentage_steps = {
+            "start": 0,
+            "process": 10,
+            "build_output": 95,
+            "complete": 100,
+        }
+
+        inputs = [
+            LiteralInput(
+                "gcm_file",
+                "GCM NetCDF file",
+                abstract="GCM simulations in NetCDF format",
+                data_type="string",
+            ),
+            LiteralInput(
+                "obs_file",
+                "Observations NetCDF file",
+                abstract="high-res gridded historical observations",
+                data_type="string",
+            ),
```

Comment:
Would it be possible for us to pass in files from the thredds server for this process? If the answer is yes I think we will need to adjust the available formats for these options.

Reply:
Passing the OPeNDAP paths will pass, but it won't have access to local paths. change these to `ComplexInput`.